db/migrate/20150713143226_add_unique_to_operatingsystems_title.rb

```diff
@@ -0,0 +1,17 @@
+class AddUniqueToOperatingsystemsTitle < ActiveRecord::Migration
+  def up
+    duplicities = Operatingsystem.all.group_by(&:title).values.keep_if { |ary| ary.length > 1}
+    duplicities.each do |systems|
+      systems.each do |os|
+        os.description = os.fullname
```

Comment:
This could inadvertently override `os.description`, what is this double loop supposed to accomplish?